src/modules/poweraccent/PowerAccentKeyboardService/KeyboardListener.h

```diff
@@ -1,7 +1,7 @@
 ﻿#pragma once
 
 #include "KeyboardListener.g.h"
-
+#include <mutex>
```

Comment:
duplicate include. there is one 2 lines below :)

Reply:
I've added a commit to address the feedback. Thank you!